astropy/coordinates/builtin_frames/utils.py

```diff
@@ -16,15 +16,12 @@
 from astropy.utils.exceptions import AstropyWarning
 
 
-# The UTC time scale is not properly defined prior to 1960, so Time('B1950',
```

Comment:
Removed the comment since it is now basically consistent, though note that I just went for `tt` for `B1950` too.